### ISSUE TITLE
Crouton not displaying

### ISSUE THREAD

**Issue by californiasteve** [bmlt-enabled/crouton#131]
I have two sites this isn't working on. They're two separate servers also. BMLT servers are on the same server.
https://naitaliadev.com/tabbed-meeting-search/
[bmlt_tabs show_map="1" max_zoom_level="6" has_areas="1" show_distance="1"]  
https://na-italia.info/main_server
https://hillcountryna.org/bmlt-table-test/
[bmlt_tabs service_body_parent="1018,1054,1033,1048,1029,1068,1042" maps="1" has_areas="1" show_distance="1"]
https://texasoklahomana.org

**Comment by radius314** [comment]
the root server is throwing a 500 error on this query...

```curl --head -g "https://texasoklahomana.org/main_server/client_interface/json/?switcher=GetSearchResults&data_field_key=location_postal_code_1,duration_time,start_time,weekday_tinyint,service_body_bigint,longitude,latitude,location_province,location_municipality,location_street,location_info,location_text,formats,format_shared_id_list,comments,meeting_name,location_sub_province,worldid_mixed,root_server_uri&services[]=1018&services[]=1054&services[]=1033&services[]=1048&services[]=1029&services[]=1068&services[]=1042&recursive=1&sort_key=time"

HTTP/1.1 500 Internal Server Error
Set-Cookie: PHPSESSID=jgj0uo9m76dh083u0jisco1n55; path=/
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma: no-cache
Content-Type: text/html
Date: Sat, 29 Jun 2019 02:07:32 GMT
Server: LiteSpeed
Vary: User-Agent
Alt-Svc: quic=":443"; ma=2592000; v="35,39,43,44"
Connection: Keep-Alive
```

**Comment by radius314** [comment]
The na-italia issue is totally unrelated, and has a rendering issue that I'm looking into.

**Comment by radius314** [comment]
On the Italy one can you try to change `max_zoom_level` to `map_max_zoom`?

**Comment by californiasteve** [comment]
No change

**Comment by californiasteve** [comment]
I'm seeing this class being loaded "hide" and on other installations I have this says "show"
![image](https://user-images.githubusercontent.com/31661648/60384797-e9f4c980-9a47-11e9-85ab-6b47d4c627c9.png)

**Comment by radius314** [comment]
https://naitaliadev.com/tabbed-meeting-search/ seems to have `[bmlt]` on it, and crouton isn't on the page anymore.  Also I'm noticing that the page is bouncing, not sure why.

**Comment by radius314** [comment]
Are you able to pull the error log on the root server for this request, resulting in an HTTP 500?

https://texasoklahomana.org/main_server/client_interface/jsonp/?switcher=GetSearchResults&data_field_key=location_postal_code_1,duration_time,start_time,weekday_tinyint,service_body_bigint,longitude,latitude,location_province,location_municipality,location_street,location_info,location_text,formats,format_shared_id_list,comments,meeting_name,location_sub_province,worldid_mixed,root_server_uri&services[]=1018&services[]=1054&services[]=1033&services[]=1048&services[]=1029&services[]=1068&services[]=1042&recursive=1&sort_key=time&callback=jQuery112408337667706427289_1562079191310&_=1562079191311

**Comment by californiasteve** [comment]
Can't do it until later.

⁣Get BlueMail for Android ​

**Comment by tkasis** [comment]
From the server:

> [texasoklahomana@host json]$ tail error_log
> [03-Jul-2019 00:13:12 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 82 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 246
> [03-Jul-2019 00:13:38 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 736 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 246
> [03-Jul-2019 00:26:57 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 72 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_meeting.class.php on line 1623
> [03-Jul-2019 02:09:06 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 83 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 251
> [03-Jul-2019 02:09:43 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 83 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 251
> [03-Jul-2019 02:22:25 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 83 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 251
> [03-Jul-2019 02:26:56 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 72 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_meeting.class.php on line 1613
> [03-Jul-2019 04:26:55 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 72 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_meeting.class.php on line 1623
> [03-Jul-2019 06:26:57 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 82 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 246
> [03-Jul-2019 08:26:55 UTC] PHP Fatal error:  Allowed memory size of 33554432 bytes exhausted (tried to allocate 736 bytes) in /home/texasoklahomana/public_html/main_server/server/classes/c_comdef_dbsingleton.class.php on line 246

Changed PHP version to 7.3 with 256M limit

**Comment by radius314** [comment]
Hill Country is loading now.

**Comment by radius314** [comment]
@californiasteve when you get a chance can you confirm that hill country is no longer an issue.

**Comment by californiasteve** [comment]
It's working on the https://hillcountryna.org fine but still getting the <div id="bmlt-tabs" class="bmlt-tabs hide"> This is on a different server the hillcountryna

**Comment by radius314** [comment]
Some part of your message seems to be cut off.

**Comment by californiasteve** [comment]
div id="bmlt-tabs" class="bmlt-tabs hide"

**Comment by radius314** [comment]
Are we still taking about the Italian test?

**Comment by radius314** [comment]
This looks like an issue with handlebars.js being loaded from another plugin.  Can you share the list of plugins installed as well as the theme you are using (I think it's Colormag).

**Comment by radius314** [comment]
My guess is that it's the eventON plugin.

**Comment by californiasteve** [comment]
![image](https://user-images.githubusercontent.com/31661648/60776628-38aefd00-a0f3-11e9-9481-9aade75f9f84.png)
![image](https://user-images.githubusercontent.com/31661648/60776633-43699200-a0f3-11e9-8b8b-178039855ad4.png)

**Comment by radius314** [comment]
way ahead of you

**Comment by californiasteve** [comment]
lol, I disabled eventon and it's still not rendering

**Comment by radius314** [comment]
Please try this version of crouton https://archives.bmlt.app/crouton/crouton-build197-df89b65f8d7b5edbce46fda534106ec7a92da41a.zip

**Comment by californiasteve** [comment]
That works now.

**Comment by radius314** [comment]
Ok cutting a release, look for this fix in crouton 3.2.1.

**Comment by californiasteve** [comment]
Having same problem using the url call to page with just the [bmlt_tabs]
https://naitaliadev.com/tabbed-map-search/?root_server=https://na-italia.info/main_server/bmlt_server&service_body=2&this_title=Brescia%20Ara&meeting_count=1&group_count=1&has_states=1&show_map=1
